pkg/proto/datadog/test/v1/api.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package datadog.test.v1;
+
+import "google/protobuf/empty.proto";
+
+option go_package = "pkg/proto/test2"; // golang
```

Comment:
let's rename the package to be a valid name

Reply:
you need to change the value here, otherwise any future generation of the protobuf will make a diff.